### PR TITLE
New Attributes & Form attributes #13646

### PR DESCRIPTION
**Pull request by emiliodeg**
Implemented in Form
Changed CHANGELOG
Added tests

Hello!

Type: new feature
Link to issue: #13646

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I updated the CHANGELOG

Small description of change:
I created a new interface for HTML Attributes and I implemented in Form
I replaced the action property insteand of attributes
I writed tests
Thanks

### CODE REVIEW

File: tests/integration/Forms/FormCest.php [hunk 1]
```diff
@@ -354,31 +602,157 @@ public function testElementMessages(IntegrationTester $I)
...
+                            'telephone',
+                            'PresenceOf',
+                            0
+                        ),
+                    ]
+                ),
+            ],
+        ];
+        $actual   = $form->getMessages(true);
+        $I->assertEquals($expected, $actual);
```
Comment (sergeyklay) [inline tests/integration/Forms/FormCest.php]: Phalcon\Messages\Messages Object (...) does not match expected type "array"

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by niden** [comment]
@emiliodeg If you can rebase this and check on the tests that would be great. Thanks

**Comment by emiliodeg** [comment]
How can I rebase? I just only see merge

**Comment by Jurigag** [comment]
In git.

**Comment by niden** [comment]
@emiliodeg I will rebase that for you if you don't know. Have a look at Travis so that this passes and then I will merge it.

**Comment by niden** [comment]
Closing in favor of https://github.com/phalcon/cphalcon/pull/14082

**Pull request closed without merging**